patches/ubuntu/build/install-gpu.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+set -eo pipefail
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/install.sh
+
+# NVIDIA CUDA drivers
+DEBIAN_FILE="cuda-keyring_1.1-1_all.deb"
+REPO_URL="https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2404/x86_64/$DEBIAN_FILE"
+GPG_KEY="/usr/share/keyrings/cuda-archive-keyring.gpg"
+REPO_PATH="/etc/apt/sources.list.d/cuda-ubuntu2404-x86_64.list"
```

Comment:
The envs `REPO_URL` and `REPO_PATH` contain slugs/paths specific to Ubuntu 24.04, so I'm not sure you'd want to use it as is for back porting to 22.04. Perhaps you may want to parametrize that?